### PR TITLE
feat: add about page

### LINKED ISSUE #1: feat: Add an About page
The footer link of the about page points to the home page.

It's nice to have an about page letting the community know about the project. 

Read this article to get inspiration about the content of the page: https://www.adxy.dev/projects/chesskhelo

A good example of an about page would be - https://lichess.org/about

Interested to add this? Comment & I'll assign this to you.

### PR DESCRIPTION
**Pull request by cjiayu03**

### CODE REVIEW

File: pages/about.jsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      originated when its creator, Adarsh Bhadauria, was looking for a new challenge and decided to mess with CSS</p>
+    
+      <p>With the use of CSS, Adarsh was able to create a chessboard pattern, which served as the foundation 
+      for the development of Chesskhelo. As the project progressed, a plethora of additional features and 
+      functionality were added to the server, including support for online multiplayer games, in-game chat and touch devices </p>
+  
+      <p>Not only Chesskhelo is constantly evolving, but it is also community-driven. The development team 
+      is always open to feedback and suggestions from users, and they actively seek out new ideas and features 
+      to improve the overall playing experience. If you have an idea for a new feature or have found a bug, you 
+      can contact the development team by raising an issue request on our Github repository<p>
```
Comment (adxy) [inline pages/about.jsx]:
```suggestion
      can contact the development team by raising an issue request on our GitHub repository</p>
```

File: pages/about.jsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+        
+      <p>Additionally, Chesskhelo welcomes community contributions. The open-source nature of the project means 
+      that anyone can contribute to the codebase and help to improve the server. Whether you are an experienced 
+      developer or just starting out, you can make a difference by contributing to Chesskhelo. If you are interested 
+      in getting involved with the project, you can check out the Github repository for more information on how to contribute<p>
```
Comment (adxy) [inline pages/about.jsx]:
```suggestion
      in getting involved with the project, you can check out the GitHub repository for more information on how to contribute</p>
```

**Review by adxy — CHANGES_REQUESTED** [review]
Hey @c00kie123, I see that you're missing on closing some "p" tags, I've highlighted those in the code. 

Also, I think you've not setup the code and run it on the local machine. Please do it to make sure the page is linted successfully. 

Read the Readme.md to know how to set it up locally. You might also have to install two vs code plugins - eslint & prettier

Let me know if you need help with anything

**Comment by cjiayu03** [comment]
Hey, I have made the changes and it is working in my environment

**Comment by adxy** [comment]
Hey @c00kie123. I will review this in the next 24hrs. I am a little occupied with something today. Thanks for your patience. 😀

**Review by adxy — APPROVED** [review]
There are some formatting issues, I guess you've not successfully set up prettier plugin for vs code. I'll go ahead and accept this as this can be fixed later. Good job, thanks for your contribution! .✨

**Comment by c00kie123** [comment]
Hey, what are the formatting issues? I will try it on my environment

**Pull request merged**